nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -569,6 +569,7 @@ in
         atomix
         five-or-more
         four-in-a-row
+        gnome-2048
```

Comment:
There does not appear to be an official gnome-games set, all games are just in world so it is fine here:

https://gitlab.gnome.org/GNOME/gnome-build-meta/-/blob/9eba7b5cd26e22df15b254ca41ee3219b1059c56/elements/world.bst#L21